README.md

```diff
@@ -625,6 +626,14 @@ descriptors will fail. Descriptors sent to Memcache should not contain whitespac
 When using multiple memcache nodes in `MEMCACHE_HOST_PORT=`, one should provide the identical list of memcache nodes
 to all ratelimiter instances to ensure that a particular cache key is always hashed to the same memcache node.
 
+# Custom headers
```

Comment:
It'd be good to link to the official Envoy specs in this section so its more clear what is being set

Reply:
definitly, will add